### PR TITLE
Add jquery.complexify.js v0.5.1

### LINKED ISSUE #5603: [Request] jQuery Complexify
**Library name:** jQuery Complexify
**Git repository url:** https://github.com/danpalmer/jquery.complexify.js/
**npm package url(optional):** 
**License(s):** WTFPL v2
**Official homepage:** https://www.danpalmer.me/jquery-complexify
**Wanna say something? Leave message here:**
Nope :+1: 

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
@PeterDaveHello Please help me check it.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/danpalmer/jquery.complexify.js
 * Official website (optional, not the repository): https://www.danpalmer.me/jquery-complexify
 * NPM package url (optional): 
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 634
   - Count of watchers: 54
   - Count of forks: 178
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with git auto-updated config added close #5603 cc @danpalmer

### CODE REVIEW

File: ajax/libs/jquery.complexify.js/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/danpalmer/jquery.complexify.js.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:danpalmer/jquery.complexify.js.git",
+    "basePath": "",
+    "files": [
+      "**/*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.complexify.js/package.json]: Should not use this kind of wildcard in non-dist folder.

Comment (PeterDaveHello) [inline ajax/libs/jquery.complexify.js/package.json]: You should use `jquery.complexify*` in this case.

**Comment by x09326** [comment]
@PeterDaveHello I have already modified it, please help me check it. Thank you.

File: ajax/libs/jquery.complexify.js/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.complexify.js",
+  "filename": "jquery.complexify.min.js",
+  "version": "0.5.1",
+  "description": "JavaScript library for inform users of password strength",
+  "homepage": "https://github.com/danpalmer",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.complexify.js/package.json]: The homepage is `http://danpalmer.me/jquery-complexify`

Comment (x09326) [inline ajax/libs/jquery.complexify.js/package.json]: @PeterDaveHello I already fixed it. Please help me check it. Thank you!

Comment (PeterDaveHello) [inline ajax/libs/jquery.complexify.js/package.json]: Didn't see your fix.

File: ajax/libs/jquery.complexify.js/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.complexify.js",
+  "filename": "jquery.complexify.min.js",
+  "version": "0.5.1",
+  "description": "JavaScript library for inform users of password strength",
+  "homepage": " http://danpalmer.me/jquery-complexify",
```
Comment (x09326) [inline ajax/libs/jquery.complexify.js/package.json]: Sorry, I misread it and now it is okay.
@PeterDaveHello please help me check other part. Thank you.

File: ajax/libs/jquery.complexify.js/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.complexify.js",
+  "filename": "jquery.complexify.min.js",
+  "version": "0.5.1",
+  "description": "JavaScript library for inform users of password strength",
+  "homepage": "http://danpalmer.me/jquery-complexify",
```
Comment (x09326) [inline ajax/libs/jquery.complexify.js/package.json]: Sorry, I misread it last time. It is okay now.
@PeterDaveHello Please help me check other part. Thank you!

**Comment by x09326** [comment]
The author sends this http://www.wtfpl.net/txt/copying/ for the license information.
Should I add the license as  WTFPL in the package.json ?
@PeterDaveHello

**Comment by PeterDaveHello** [comment]
Okay, it'll be great, looks like the WTFPL v2 is just "WTFPL" on SPDX:

https://spdx.org/licenses/WTFPL.html#licenseText

cc @cdnjs/intern2

**Comment by x09326** [comment]
@PeterDaveHello I have already added the license.
Please take a look. Thanks.

**Comment by PeterDaveHello** [comment]
Please also add `close #6000` in your commit message next time, thanks.

**Comment by x09326** [comment]
Got it, thanks.

**Pull request merged**